test/test.sh

```diff
@@ -391,7 +477,7 @@ fi
 TEST_TMPDIR=/tmp/yugabyte-bash-common-test.$$.$RANDOM.$RANDOM.$RANDOM
 mkdir -p "$TEST_TMPDIR"
 
-trap cleanup EXIT
+#trap cleanup EXIT
```

Comment:
Does  this  need to be uncommented before landing?

Reply:
!! yes !!